numpy/distutils/command/build_ext.py

```diff
@@ -270,7 +270,7 @@ def run(self):
             for ext in self.extensions
         }
         for pkg_root in pkg_roots:
-            shared_lib_dir = os.path.join(pkg_root, '.libs')
+            shared_lib_dir = os.path.join(pkg_root, 'core')
```

Comment:
may be overkill, not sure, I think @mattip mostly wanted me to focus in on `extra_dll_dir`; may have to dissect a bit more to find the necessary + sufficient change set.

Reply:
I guess we may want `.libs` to disappear though if we go through with a change like this.

Reply:
this change looks fine